### ISSUE TITLE
Unable to sync files containing ':' in name

### ISSUE THREAD

**Issue by rahra** [owncloud/android#962]
The Onwcloud Android app fails to download files from the Owncloud server if there is a ':' in the file name. The GET request looks like the following example:
GET /owncloud/remote.php/webdav/Notes/2014-11-21%2014%3A38%3A10.txt HTTP/1.1
The HTTP server responds with 200 but the app then says 'Download failed'. If the file is renamed by replacing all ':'s to some other character, the download works.

**Comment by davivel** [comment]
We are not able to reproduce it. What are your device model and version? I'm suspecting of a problem with the ':' in the local file system. Could you check if you can rename a local file to include a ':' with a regular file browser?

Thanks.

**Comment by rahra** [comment]
Sorry for filing this report because it is definitely a matter of the local file system. On the SDcard there is FAT by default which doesn't allow ':' in the file names.

FYI what I did:
I used the ['MyOwnNotes' app](https://f-droid.org/repository/browse/?fdfilter=myownnotes&fdid=org.aykit.MyOwnNotes) which creates file names with ':' in it but on the server. The server of course allows such names because it usually uses a modern file system (which is UFS with FreeBSD in my case) and not FAT.
But when I try to download those files with the OwnCloud client then it fails to create the file locally.

I suggest to make the error message more clear because it says 'Download failed' but actually the download is ok but the file creation fails.
